streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -1776,27 +1229,6 @@ public void process(final Record<String, String> record) {
         }
     }
 
-    @SuppressWarnings("deprecation") // Old PAPI. Needs to be migrated.
```

Comment:
Seems we should not remove this (it's not `@Deprecated`) and comment says `Need to be migrated`, but rather rewrite to for the new processor API? Or is it unused now?

Reply:
@pegasas -- i don't see any reply on this comment (3re one)